TestModels/Documentation/Model/documentation.smithy

```diff
@@ -0,0 +1,92 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+namespace simple.documentation
+
+/// A service that supports the operation of getting things.
+@aws.polymorph#localService(
+  sdkId: "SimpleDocumentation",
+  config: SimpleDocumentationConfig,
+)
+service SimpleDocumentation {
+  version: "2021-11-01",
+  resources: [ Widget ],
+  operations: [ GetThing ],
+  errors: [ CouldntGetTheThingError ],
+}
+
+/// Fancy configuration things to make getting things even thingier.
+structure SimpleDocumentationConfig {}
+
+/// A thing that you can get from the service.
+structure Thing {
+  /// The name of the thing.
+  name: String
+}
+
+/// Call this to get a thing.
+operation GetThing {
+  input: GetThingInput
+  output: GetThingOutput
+}
+
+/// Inputs for getting a thing.
+structure GetThingInput {
+  /// The name of the thing to get, obviously.
+  @required
+  name: String
+}
+
+/// Outputs for getting a thing.
+structure GetThingOutput {
+  /// The thing that you just got.
+  @required
+  thing: Thing
+}
+
+/// A different kind of thing you can get.
```

Comment:
Where in the spec does it say `///` is a comment? It clearly is, but I can't find the spec.

Reply:
https://smithy.io/2.0/spec/idl.html#documentation-comment

Reply:
What! Why?
That... ugh...
At least one of of these `///` should be replaced with the actual trait instance.
https://smithy.io/2.0/spec/documentation-traits.html#smithy-api-documentation-trait

Maybe I am crazy, but I hope that our team always uses the actual trait,
instead of `///`.

`///` is too easy to confuse with `//` or `//#`,
which are just regular comments
and duvet comments respectively.

Reply:
I admit I AM disappointed that the VS Code Smithy extension doesn't give `///` comments a different color or something. :P